lib/ibandit/iban.rb

```diff
@@ -362,6 +363,13 @@ def valid_nz_details?
       valid_modulus_check_branch_code?
     end
 
+    def valid_ca_details?
+      return true unless country_code == "CA"
```

Comment:
maybe we don't need this guard because we reach here after `case country_code; when "CA"`?

Reply:
just being doubly sure in case this method gets called independently (which is highly unlikely, I know). wouldn't worry too much about it in this case as:
- the additional check isn't exactly expensive and
- it stays consistent with the existing `valid_*_details?` methods

Reply:
guessed so, thanks for confirming.